test_scaffolding_go.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+set -euxo pipefail
+
+mkdir -p cpp-tests/build
+pushd cpp-tests/build
+cmake -DCMAKE_BUILD_TYPE=Debug ..
+make uniffi_fixtures
+
+popd
+
+SCRIPT_DIR="${SCRIPT_DIR:-$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )}"
+ROOT_DIR="$SCRIPT_DIR"
+
+GENERATOR_DIR="$ROOT_DIR/3rd-party/uniffi-bindgen-go"
+BINDINGS_DIR="$GENERATOR_DIR/binding_tests"
+BINARIES_DIR="$ROOT_DIR/cpp-tests/build"
+
+pushd $GENERATOR_DIR
+cargo build
+./build_bindings.sh
+
+
+# We add the skip section, that skips a few chronological test functions
+# due to C++ chrono library not supporting large timestamp values
+pushd $BINDINGS_DIR
+LD_LIBRARY_PATH="${LD_LIBRARY_PATH:-}:$BINARIES_DIR" \
+	CGO_LDFLAGS="-luniffi_fixtures -L$BINARIES_DIR -lm -ldl" \
+	CGO_ENABLED=1 \
+	go test \
+	binding_tests.go \
```

Comment:
I would to invert this list of tests. Instead of manually listing the needed tests, list the tests that are not needed. This way new test files would not be forgotten with subsequent versions. The way to accompilish this would be to list files `$BINDINGS_DIR ` and remove the offending files. Listing the files is trivial in bash, `echo *.go`, but the removing part seems more complicated.